bandit/formatters/xml.py

```diff
@@ -27,7 +27,8 @@
     <testsuite name="bandit" tests="1"><testcase
     classname="examples/yaml_load.py" name="blacklist_calls"><error
     message="Use of unsafe yaml load. Allows instantiation of arbitrary
-    objects. Consider yaml.safe_load().&#10;" type="MEDIUM">Test ID: B301
+    objects. Consider yaml.safe_load().&#10;" type="MEDIUM"
+    more_info="https://docs.openstack.org/developer/bandit/">Test ID: B301
```

Comment:
I'd rather not have the more_info link example be to the openstack docs.  The docs are now hosted on https://bandit.readthedocs.io/en/latest/

Reply:
will fix